.github/ISSUE_TEMPLATE/cve_report.md

```diff
@@ -0,0 +1,16 @@
+---
+name: CVE Finding Report
```

Comment:
I just want to be sure people don't confuse reporting an existing CVE in some library we use, to a new founding in ingress-nginx and publicly announcing it with an open issue :)

Can we add some huge notice saying "if you found something that impacts directly ingress-nginx and is not a public CVE yet, please reach security@kubernetes.io"?

Reply:
done